traefik/tests/requirements-config_test.yaml

```diff
@@ -14,6 +14,7 @@ tests:
       image:
         repository: traefik/traefik
         tag: experimental-v3.0
+      versionOverride: v3.1
```

Comment:
Your changes shouldn't break this test.  We need to keep the experimental compatibility.